docs/content/en/docs/getting-started/observability/_index.md

```diff
@@ -1,6 +1,244 @@
 ---
 title: Standardize access to observability data
-description: Learn how the Keptn Lifecycle Toolkit standardizes access to observability data.
+description: Learn how the Keptn Lifecycle Toolkit provides observability for Kubernetes deployments
 weight: 45
 ---
 
+The Keptn Lifecycle Toolkit (KLT) makes any Kubernetes deployment observable.
+You can readily see why a deployment takes so long or why it fails,
+even when using multiple deployment tools.
+Keptn provides a Keptn application resource
+that aggregates multiple Workloads into a single resource
+that can be monitored.
```

Comment:
I would maybe say it introduces a concept of an application, which is natively not part of the Kubernetes and therefore is able to create a layer upon running workloads